test/unit/stPacket/verification/verifySTPacketFactory.spec.js

```diff
@@ -189,4 +195,64 @@ describe('verifySTPacketFactory', () => {
     expect(verifyContractMock).to.have.not.been.called();
     expect(verifyDocumentsMock).to.have.been.calledOnceWith(stPacket, userId);
   });
+
+  it('should execute data triggers and properly add errors to the final result', async () => {
```

Comment:
"should return an invalid result if data triggers return error" and move it above